app/controllers/api/v2/people_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    class PeopleController < ApiController
+      before_action :validate_include_params, only: :index
+
+      def index
+        people = ::V2::People::Finder.new(filter_params).call
+
+        paginate people, include: included_relationships, each_serializer: ::V2::PersonSerializer
+      end
+
+    private
+
+      PERMITTED_FILTER_PARAMS = %i[police_national_computer criminal_records_office prison_number].freeze
+
+      def filter_params
+        params.fetch(:filter, {}).permit(PERMITTED_FILTER_PARAMS).to_h
+      end
+
+      def included_relationships
+        IncludeParamHandler.new(params).call
+      end
+
+      def validate_include_params
+        ::V2::People::IncludeParamsValidator.new(included_relationships).tap do |validator|
```

Comment:
Unfortunately if `include` has values that are not in relationships it should return a `400` rather than a validation error `422` as per the specification: https://jsonapi.org/format/#fetching-includes

Reply:
I think we should keep consistent with our way of rendering errors: https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/5bba6f774d9a87ac69b07418d454d1f4d48a47f2/app/controllers/api_controller.rb#L11-L17

Reply:
I remember that I had a conflict with another  `render_bad_request_error`,
we are already using that for ParameterMissin, 
We'll have a look

Reply:
We agreed that we are going to handle this on the next PRs, since those are relevant to this (post and patch)